### ISSUE TITLE
env.Set() is not reflecting values in config.yml

### ISSUE THREAD

**Issue by nwlucas** [nwlucas/proj-cli#2]
When `proj-cli debug` is called the resulting output shows the that Cfg.config is empty as shown below:

```shell
Debug Information.

	Environment:


[]string{"GREP_COLOR=1;33", "XDG_VTNR=7", "ORBIT_SOCKETDIR=/tmp/orbit-nwilliams", "SSH_AGENT_PID=12536", "XDG_SESSION_ID=146", "TODO_SRC_DIR=/home/nwilliams/.bash_it/plugins/available/todo", "TERMINATOR_UUID=urn:uuid:d0aadbc1-482a-4205-9cb7-53ef2b92a161", "COMP_WORDBREAKS= \t\n\"'><;|&(", "GIO_LAUNCHED_DESKTOP_FILE_PID=15482", "PYENV_ROOT=/home/nwilliams/.pyenv", "GPG_AGENT_INFO=/run/user/1000/keyring/gpg:0:1", "NGINX_PATH=/opt/nginx", "TERM=xterm", "SHELL=/bin/bash", "HISTSIZE=5000", "GJS_DEBUG_OUTPUT=stderr", "GO15VENDOREXPERIMENT=1", "GIT_HOSTING=git@gitlab.nwlnexus.net", "WINDOWID=44040196", "GJS_DEBUG_TOPICS=JS ERROR;JS LOG", "NVM_DIR=/home/nwilliams/.nvm", "USER=nwilliams", "TODOTXT_DEFAULT_ACTION=ls", "BASH_IT_THEME=envy", "RBENV_ROOT=/usr/local/rbenv", "SCM_CHECK=true", "SSH_AUTH_SOCK=/run/user/1000/keyring/ssh", "FAB_COMPLETION_CACHE_TASKS=true", "SESSION_MANAGER=local/triarii:@/tmp/.ICE-unix/12199,unix/triarii:/tmp/.ICE-unix/12199", "USERNAME=nwilliams", "AUTOFEATURE=true", "BASH_IT=/home/nwilliams/.bash_it", "GIO_LAUNCHED_DESKTOP_FILE=/usr/share/applications/terminator.desktop", "LSCOLORS=Gxfxcxdxdxegedabagacad", "PATH=/usr/local/rbenv/shims:/home/nwilliams/projects/bin:/usr/local/rbenv/shims:/home/nwilliams/.rbenv/bin:/home/nwilliams/.pyenv:./node_modules/.bin:/home/nwilliams/projects/bin:/home/nwilliams/.npm/bin:/usr/local/rbenv/shims:RBENV_ROOT/bin:/usr/local/rbenv/shims:/usr/local/rbenv/bin:/usr/local/bin:/usr/bin:/bin:/usr/local/games:/usr/games:/usr/local/go/bin:/opt/nginx/sbin:/home/nwilliams/.gem/ruby/2.1.0/bin:/home/nwilliams/.bash_it/plugins/available/todo:/home/nwilliams/projects/gocode/bin:/home/nwilliams/projects/gocode/bin", "DESKTOP_SESSION=default", "PWD=/home/nwilliams/projects/gocode/src/github.com/nwlucas/proj-cli", "LANG=en_US.UTF-8", "GDM_LANG=en_US.UTF-8", "NODE_PATH=/home/nwilliams/.npm/lib/node_modules:/usr/lib/nodejs:/usr/lib/node_modules:/usr/share/javascript", "FAB_COMPLETION_CACHED_TASKS_FILENAME=.fab_tasks~", "GDMSESSION=default", "RBENV_SHELL=bash", "HISTCONTROL=erasedups", "TODO_DIR=/home/nwilliams/.bash_it/custom", "CINNAMON_VERSION=2.2.16", "SHLVL=1", "XDG_SEAT=seat0", "HOME=/home/nwilliams", "GOROOT=/usr/local/go", "GNOME_DESKTOP_SESSION_ID=this-is-deprecated", "XDG_SESSION_DESKTOP=default", "LOGNAME=nwilliams", "XDG_DATA_DIRS=/usr/share/gnome:/usr/local/share/:/usr/share/", "DBUS_SESSION_BUS_ADDRESS=unix:abstract=/tmp/dbus-IALRXw1ICL,guid=a0ee55e57368d98be5dae023564e8794", "GOPATH=/home/nwilliams/projects/gocode", "TODO=t", "WINDOWPATH=7:2:3:7:7", "IRC_CLIENT=irssi", "XDG_RUNTIME_DIR=/run/user/1000", "DISPLAY=:0", "POSTGRES_BIN=/usr/lib/postgresql/9.4/bin", "XDG_CURRENT_DESKTOP=X-Cinnamon", "PGVERSION=9.4.5", "COLORTERM=gnome-terminal", "XAUTHORITY=/var/run/gdm3/auth-for-nwilliams-VqmMuQ/database", "_=/home/nwilliams/projects/gocode/bin/proj-cli", "OLDPWD=/home/nwilliams/projects/gocode/src/github.com/nwlucas"}

	Go Specific:


env.runEnv{GoOs:"linux", GoRoot:"/usr/local/go", GoVersion:"go1.5.1", NumCPU:4}

	Config:


&cfg.Config{
    keyDelm:     ".",
    configName:  "config",
    configFile:  "",
    configType:  "yml",
    configPaths: {"/home/nwilliams/.proj-cli", "/home/nwilliams/projects/gocode/src/github.com/nwlucas/proj-cli"},
    config:      {
    },
    defaults: {
    },
    overrides: {
    },
    aliases:        {},
    typeByDefValue: false,
}

	Config using Spew:


(*cfg.Config)(0xc82008e400)({
 keyDelm: (string) (len=1) ".",
 configName: (string) (len=6) "config",
 configFile: (string) "",
 configType: (string) (len=3) "yml",
 configPaths: ([]string) (len=2 cap=2) {
  (string) (len=25) "/home/nwilliams/.proj-cli",
  (string) (len=63) "/home/nwilliams/projects/gocode/src/github.com/nwlucas/proj-cli"
 },
 config: (map[string]interface {}) {
 },
 defaults: (map[string]interface {}) {
 },
 overrides: (map[string]interface {}) {
 },
 aliases: (map[string]string) {
 },
 typeByDefValue: (bool) false
})
```

**Comment by nwlucas** [comment]
Fixed in commit e98ef7

**Comment by nwlucas** [comment]
Fix inclued in SHA:e98ef795884a5c415067b23198a4af2b3d34dc8e
